apidoc/Titanium/UI/iOS/MenuPopup.yml

```diff
@@ -33,8 +33,8 @@ methods:
     summary: Hides the menu popup.
     parameters:
       - name: params
-        summary: Includes options how the menu popup should be hidden.
-        type: MenuPopupHideParams
+        summary: Includes options how the menu popup should be hidden. Introduced in SDK 5.2.0
+        type: AnimatedOptions
```

Comment:
Default was `{animated: true}`

Reply:
Yeah, I struggled with how to handle the cases where the default `animated` value is `true` vs `false`. Do we just mention it in the summary? Do I define a new sub-type of `AnimatedOptions` that just overrides the property's default value to be `true`?

Reply:
To do some follow-up: These special "types" I've been extracting are really a better match as TypeScript interfaces, not classes. Because there's no actual way to instantiate an `AnimatedOptions` type - no such thing exists. It's more about the shape of a generic JS object we expect to be passed in to the methods. As such, the conversion to TypeScript would drop any "default" values on objects like this. It's more about documenting the property names, whether they're optional/required, and if they're "readonly". So trying to set the default value on the type may actually be moot - and I may need to just be thorough in mentioning the assumed defaults on all the methods that accept it. 😢

Reply:
Just thought of another possible way to document this - set the default value on the method argument to the raw JS object that it "assumes". Old docs already do this for things like Point (i.e. `{ x: 0, y: 0 }`) We've been pretty loosey-goosey with how we handle `default` though. Not sure if we'd need more rules around it or just assume that the string value is the raw JS code/value that is assumed to be default. (How would we validate that, since I know we do have cases where we have sentences there?)

Reply:
At least we should keep this information in `summary` or `description` (same for iOS-only properties).